src/collectors/libvirtkvm/test/testlibvirtkvm.py

```diff
@@ -21,6 +22,47 @@ def setUp(self):
     def test_import(self):
         self.assertTrue(LibvirtKVMCollector)
 
+    @patch('libvirt')
```

Comment:
Someone can help here ? I need to mock the entire libvirt module...

I'm not a python guy and had never written this kind of tests :boom: